storm-core/src/jvm/org/apache/storm/utils/Utils.java

```diff
@@ -1672,11 +1672,12 @@ public void uncaughtException(Thread thread, Throwable thrown) {
      * @param key The key pointing to the value to be redacted
```

Comment:
Nit: This method now redacts values of any type, Javadoc should maybe be updated